server/src/test/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationBuilderTest.java

```diff
@@ -0,0 +1,82 @@
+package org.opentosca.toscana.plugins.cloudformation;
+
+import com.scaleset.cfbuilder.core.Module;
+import com.scaleset.cfbuilder.core.Parameter;
+import com.scaleset.cfbuilder.core.Template;
+import com.scaleset.cfbuilder.ec2.Instance;
+import com.scaleset.cfbuilder.ec2.SecurityGroup;
+import com.scaleset.cfbuilder.ec2.SecurityGroupIngress;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+
+public class CloudFormationBuilderTest extends Module {
+    
+    @Test
+    public void simpleTest() throws Exception {
```

Comment:
What does this test do?

Please consider reading [This](https://github.com/StuPro-TOSCAna/TOSCAna/blob/master/docs/dev/server-testing.md). Just to quote the important part:

>The name of a test case (method) within a class should express the purpose of the test in camel-case as concise as possible.